Extension/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # C/C++ for Visual Studio Code Changelog
 
+## Version 1.27.3: September 3, 2025
+### Enhancement
+* Show a warning when too many files are processed in a workspace. [#10828](https://github.com/microsoft/vscode-cpptools/issues/10828)
+
+### Bug Fixes
```

Comment:
* Fix input delays when editing `c_cpp_properties.json`. [#13591](https://github.com/microsoft/vscode-cpptools/issues/13591)

Reply:
@bobbrow Ah, yeah, I forgot to review the TypeScript changes, I also forgot https://github.com/microsoft/vscode-cpptools/pull/13877